appinventor/components/src/com/google/appinventor/components/runtime/Spinner.java

```diff
@@ -56,6 +56,7 @@
   private YailList items = new YailList();
   private int oldAdapterCount;
   private int oldSelectionIndex;
+  private boolean isEnabled;
```

Comment:
Do we need to store a separate state w.r.t. the spinner widget itself? It might be more efficient to defer to the underlying view's state.

Reply:
I think no, we can get and set the value directly using `view.setEnabled(value)` and `view.isEnabled()`.

but other properties store a separate state so that's why I added it, okay I will remove this extra separate state.